packages/core/src/utils/paths.ts

```diff
@@ -154,6 +154,38 @@ export function escapePath(filePath: string): string {
   return result;
 }
 
+/**
+ * Normalizes Windows-style paths by converting backslashes to forward slashes.
+ * Preserves backslashes that are used to escape shell special characters.
+ * @param filePath The path to normalize.
+ * @returns The normalized path with forward slashes.
+ */
+export function normalizePath(filePath: string): string {
```

Comment:
could be worth reviewing if this is right. would recommend deep research to figure out what the SOTA is for this.